uk.co.compucorp.civicrm.hrsampledata/CRM/HRSampleData/Importer/AbsencePeriod.php

```diff
@@ -13,7 +13,14 @@ public function __construct() {
   /**
    * {@inheritdoc}
    */
-  protected function operate(array $row) {
+  public function visit(array $row) {
+    $this->importRecord($row);
```

Comment:
I see that for all the Importers, the implementation of the visit() method is exactly the same. In that case, I think it would be better to move it CRM_HRSampleData_CSVImporterVisitor and avoid the duplication.